paasta_tools/setup_kubernetes_cr.py

```diff
@@ -85,21 +134,27 @@ def main() -> None:
         logging.basicConfig(level=logging.INFO)
 
     kube_client = KubeClient()
+    if args.output_to_std:
```

Comment:
might as well be an `if` `else` I guess?

Reply:
in that  case `KubeClient()` instantiation would be duplicated